w-books/src/App/screens/Login/layout.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './styles.css';
+import wlogo from '../../../assets/Header/wbooks_logo.svg';
+
+function Login({ inputEmail, inputPassword, onSubmit, inputValid }) {
+  return (
+    <div className="Login">
+      <div className="Login-container">
+        <img src={wlogo} className="Login-logo" alt="WBooks" />
+        <h3 className="Login-input-label">Email:</h3>
+        <input className="Login-input" type="text" placeholder="example@wolox.com.ar" onChange={inputEmail}/>
+        <h3 className="Login-input-label">Password:</h3>
```

Comment:
We start now to extract all strings into a strings file in same folder that component... so
```
Login
   | index.js
   | layout.js
   | styles.css
   | strings.js
```
where you have centraliced all component!!! string
```
export const password = 'Pasword: ';
export const email = 'Email: '
etc..
```

and then importing this string constants into your components!

Keep calm and make this little refactor in all components